release-notes/release_notes-2.18.0.md

```diff
@@ -0,0 +1,800 @@
+# OpenSearch and OpenSearch Dashboards 2.18.0 Release Notes
+
+
```

Comment:
Missing highlights sections and the OS/OSD release notes.

Reply:
Will add once it is finalized.